lochness/redcap/__init__.py

```diff
@@ -283,7 +295,7 @@ def sync(Lochness, subject, dry=False):
             content = post_to_redcap(api_url, record_query, _debug_tup)
 
             # check if response body is nothing but a sad empty array
-            if content.strip() == '[]':
+            if content.strip() == b'[]':
```

Comment:
Can't it be just `if not content.strip()` ?

Reply:
It's an empty content represented by `b'[]'` by the `redcap API`, not boolean. So the logic above is required.